pluspy.py

```diff
@@ -774,6 +774,30 @@ def parseError(a, r):
         shortest = r
     return (False, a, r)
 
+
+
+# Handy routine for rules that simply call other rules
+# It parses s using the given rule and returns a node (name, (t, a), r)
+# where name is the type of the AST node and (t, a) the result of
+# parsing given the rule.
+#
+# If t is a "Concat" rule (sequence of other rules), then you can
+# select a subset using the select argument
+#
+# Otherwise, if select is not None, the result of the rule is
+# directly returned without adding a new AST node
```

Comment:
These comments could be moved to a docstring of function `match`, which currently has no [docstring](https://www.python.org/dev/peps/pep-0257/).